### ISSUE TITLE
Parsing SEED: 'Date is required.' Warning

### ISSUE THREAD

**Issue by bonaime** [obspy/obspy#1564]
Hi,

Each time I want to read a dataless with different periods of time, I have this annoying warning message:

```
from obspy.io.xseed import Parser
from obspy import UTCDateTime
Parser('http://geoscope.ipgp.fr/metadata/G/dataless.G.CAN.seed')
/Users/bonaime/git/obspy/obspy/io/xseed/fields.py:374: UserWarning: Date is required. warnings.warn('Date is required.', UserWarning)
```

Is there a nice way to avoid this warning ? I try that but it is not working

``` code
from obspy.io.xseed import Parser
from obspy import UTCDateTime
Parser('http://geoscope.ipgp.fr/metadata/G/dataless.G.CAN.seed').get_paz('G.CAN.00.BHZ', datetime=UTCDateTime())

```

and the result is

```
/Users/bonaime/git/obspy/obspy/io/xseed/fields.py:374: UserWarning: Date is required.
  warnings.warn('Date is required.', UserWarning)
Out[1]:
{u'digitizer_gain': 1677720.0,
 u'gain': 1.24658e+17,
 u'poles': [(-0.0120768+0.011706j),
  (-0.0120768-0.011706j),
  (-36.4684+66.8452j),
  (-36.4684-66.8452j),
  (-29.8656+380.54j),
  (-29.8656-380.54j),
  (-12145.6+0j),
  (-12145.6+0j)],
 u'seismometer_gain': 3450.0,
 u'sensitivity': 5788280000.0,
 u'zeros': [0j, 0j]}
```

**Comment by bonaime** [comment]
strict=False 
```Parser('http://geoscope.ipgp.fr/metadata/G/dataless.G.CAN.seed',strict=False).get_paz('G.CAN.00.BHZ', datetime=UTCDateTime())
```

**Comment by krischer** [comment]
Fixed in #1673
